src/gs-utils.h

```diff
@@ -63,7 +63,8 @@ GPermission	*gs_utils_get_permission	(const gchar	*id);
 gboolean	 gs_utils_strv_fnmatch		(gchar		**strv,
 						 const gchar	*str);
 GDesktopAppInfo *gs_utils_get_desktop_app_info	(const gchar	*id);
-
+gboolean	 gs_utils_rmtree		(const gchar	*directory,
```

Comment:
Why? It's aligned with the brackets above AFAICS. (and Richard uses tabs, much to my dislike but I try to be consistent with the existing code style)